controls/package_spec.rb

```diff
@@ -101,3 +101,73 @@
     its('disk_error_action') { should cmp 'SUSPEND' }
   end
 end
+
+control 'package-09' do
+  impact 1.0
+  title 'Install packages suggested by Lynis'
+  desc 'Install Debian packages libpam-tmpdir,apt-listbugs, apt-listchanges, debian-goodies, debsecan, debsums, fail2ban'
+  if os[:family] == 'debian'
+    describe package('libpam-tmpdir') do
+      it { should be_installed }
+    end
+    describe package('apt-listbugs') do
+      it { should be_installed }
+    end
+    describe package('apt-listchanges') do
+      it { should be_installed }
+    end
+    describe package('debian-goodies') do
+      it { should be_installed }
+    end
+    describe package('debsecan') do
+      it { should be_installed }
+    end
+    describe package('debsums') do
+      it { should be_installed }
+    end
+    describe package('fail2ban') do
+      it { should be_installed }
+    end
+    describe package('arpwatch') do
+      it { should be_installed }
+    end
+    describe package('arpon') do
+      it { should be_installed }
+    end
+    describe package('rkhunter') do
+      it { should be_installed }
+    end
+    describe package('acct') do
+      it { should be_installed }
+    end
+    describe package('sysstat') do
+      it { should be_installed }
+    end
+    describe package('needrestart') do
+      it { should be_installed }
+    end
+
+    cmd = inspec.command('dpkg -s apache2')
```

Comment:
inspec has a package resource for that? Also this needs to move into apache-baseline